visualization_msgs/MarkerArray markers
jsk_recognition_msgs/BoundingBoxArray boxes